### ISSUE TITLE
Interfacing with Water Connect problem

### ISSUE THREAD

**Issue by daniel-partington** [kinverarity1/python-sa-gwdata#23]
**Describe the bug**
Hi @kinverarity1 , this is a really awesome package and it has previously worked very well. It seems to the Water Connect site has changed something in the last month or two, see below:

**To Reproduce**

If trying to run the example from the how-to use, i.e.:
```
>>> import sa_gwdata
>>> wells = sa_gwdata.find_wells("5928-203 and also ULE 96")
```

**Expected behavior**
```
>>> wells
["LKW042", "ULE096"]

```

**Screenshots**
<img width="1127" alt="image" src="https://user-images.githubusercontent.com/13727531/217425800-f04811af-04ee-4a16-bb36-571d2d2b869c.png">

**Desktop (please complete the following information):**
 - OS: MacOS Monetery 12.6
 - Python version: 3.7.12
 - python-sa-gwdata version: 0.9.0

**Comment by kinverarity1** [comment]
Thanks for the heads up, will take a look

**Comment by kinverarity1** [comment]
@daniel-partington sorry about the delay in fixing this. It was caused by an unexpected change in the source data. Should be all good again now.

**Comment by kinverarity1** [comment]
The fix for this was released in v0.11

**Comment by daniel-partington** [comment]
Thank you very much @kinverarity1 ! It's working perfectly now. Much appreciated.
